normandy/base/utils.py

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
The MPLv2.0 says 

> If it is not possible or desirable to put the notice in a particular file, then You may include the notice in a location (such as a LICENSE file in a relevant directory) where a recipient would be likely to look for such a notice.
([MPLv2.0 Exhibit A](https://www.mozilla.org/en-US/MPL/2.0/#exhibit-a---source-code-form-license-notice))

Personally, I find it always undesirable to include these header blocks unless the individual file has a different license than the surrounding file. I'd vote to not include them.